app/components/Footer.tsx

```diff
@@ -41,8 +42,26 @@ const links = [
 const Footer = () => {
   return (
     <footer className="px-4 pb-6 pt-16 bg-neutral-900 text-center text-neutral-500">
-      <div className="sm:flex justify-around items-center mb-16">
-        <div className="mb-20 sm:mb-0 flex-1">
+      <div className="sm:flex flex-row-reverse justify-around items-center mb-14 md:mb-16">
```

Comment:
Perfect execution here. Tailwind to the rescue simplify some of that flex logic!